src/app/public/plugin-resources/code-examples/rich-text-display/rich-text-display-demo.component.html

```diff
@@ -0,0 +1,3 @@
+<sky-rich-text-display
+  [richText]="richText"
```

Comment:
Just out of curiosity, as a developer do you ever find examples like this confusing where we use the same name for the value that we provide for a property and for the property itself? It seems to me that something like `[richText]="demoText"` would be easier to follow, but maybe that's just because I'm coming at it from a non-dev perspective. I know we do this elsewhere and this is a less complicated code sample than other places where I've seen this sort of thing, so I'm not necessarily worried about making any change here. I'm just curious if naming values the same as the properties in code examples is actually preferred by developers for some reason. I've found it confusing at times.

Reply:
Fair point. I usually do this for 2 reasons - 1) if the property name is the same, it becomes inherently clear what the property is used for 2) Its a pattern we've used elsewhere.
I don't know if this is good practice or not, but it's worth thinking about.

Reply:
Thanks for the feedback. I'll resolve this comment since I was just curious on a general level and not suggesting changing anything in this PR.